csrankings-x.csv

```diff
@@ -197,6 +197,7 @@ Xiaopeng Hong,Harbin Institute of Technology,http://homepage.hit.edu.cn/hongxiao
 Xiaoping Chen,USTC,http://dsxt.ustc.edu.cn/zj_ywjs.asp?zzid=471,7_dDukkAAAAJ
 Xiaoping Jia,DePaul University,http://condor.depaul.edu/xjia,NOSCHOLARPAGE
 Xiaoping Zhang,Tsinghua University,http://www.cs.tsinghua.edu.cn/publish/csen/4623/2010/20101225000806223958364/20101225000806223958364_.html,p82_4bgAAAAJ
+Xiaoqian Wang 0001,Purdue University,https://sites.google.com/site/joyxqwang,I3tc214AAAAJ

```

Comment:
ECE, not CS, not listed here: https://www.cs.purdue.edu/people/faculty/index.html.